index.js

```diff
@@ -37,12 +37,11 @@ function hash(filepath, bits, format) {
 
     if (ftype.mime === 'image/png') {
       return new Promise((resolve, reject) => {
-        let png = new PNG(fdata);
-        png.decode((pixels) => {
+        new PNG().parse(fdata, (err, data) => {
```

Comment:
Since the node version is bumped, can we use async/await syntax here?

Reply:
I may be missing something, but since we're returning a Promise there's nothing to `await` here?

Reply:
You're right! `png-js` doesn't expose promise-based api so we want to wrap it...